SStorageStuff/Config/blocks.xml

```diff
@@ -10,6 +10,131 @@ blocks.xml
 <configs>
     <append xpath="/blocks">
 
+        <block name="cntCabinetBottomPlayer">
+            <property name="Class" value="SecureLoot"/> <!-- Upgraded to lockable -->
+            <property name="CreativeMode" value="Player"/>
+            <property name="Material" value="MwoodReinforced"/> <!-- strength -->
+            <property name="CustomIcon" value="cntCabinetBottomFiller"/> <!-- Because icon doesn't exist for new blockname -->
+            <property name="DescriptionKey" value="playerStorageGroupDesc"/> <!-- localisation generic -->
+            <property name="Shape" value="New"/>
+            <property name="Model" value="counter"/> 
+            <property name="Texture" value="261,345,262,345,345,345"/>
+            <property name="UseGlobalUV" value="Local"/>
+            <property name="LootList" value="212"/> <!-- Sounds like a cupboard -->
+            <property class="RepairItems"> <property name="resourceWood" value="10"/> </property>
+            <property name="DowngradeBlock" value="cntCabinetBottomPlayerInsecure"/>
+            <drop event="Destroy" count="0"/> <!-- because it will change to insecure -->
+            <property name="FilterTags" value="floot,ffurniture"/>
+            <property name="Path" value="solid"/>
+            <property name="Place" value="TowardsPlacerInverted"/>
+            <property name="IsDecoration" value="true"/>
+            <property name="EconomicValue" value="17"/>
+            <property name="EconomicBundleSize" value="20"/>
+        </block>
+
+        <block name="cntCabinetBottomPlayerInsecure">
+            <property name="Class" value="Loot"/> <!-- Upgraded to lockable -->
+            <property name="CreativeMode" value="Player"/>
+            <property name="Material" value="MwoodReinforced"/> <!-- strength -->
+            <property name="CustomIcon" value="cntCabinetBottomFiller"/> <!-- Because icon doesn't exist for new blockname -->
+            <property name="DescriptionKey" value="playerStorageGroupDesc"/> <!-- localisation generic -->
+            <property name="Shape" value="New"/>
+            <property name="Model" value="counter"/> 
+            <property name="Texture" value="261,345,262,345,345,345"/>
+            <property name="UseGlobalUV" value="Local"/>
+            <property name="LootList" value="212"/> <!-- Sounds like a cupboard -->
+            <property class="RepairItems"> <property name="resourceWood" value="10"/> </property>
+            <drop event="Destroy" count="0"/> <!-- because it will change to insecure -->
```

Comment:
This one will be destroyed - should it yield some wood?

Reply:
Left the comment there so it stands out. Admittedly a bit confusing. Left value at zero like the bookcase. Happy to apply, perhaps we can add some consistancy with the addition of the other items. 
```suggestion
            <drop event="Destroy" count="0"/> <!-- No Resources -->
```